types/mcping-js/index.d.ts

```diff
@@ -0,0 +1,36 @@
+// Type definitions for mcping-js 1.5
+// Project: https://github.com/Cryptkeeper/mcping-js
+// Definitions by: Inrix <https://github.com/Inrixia>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 4.0
```

Comment:
```suggestion
// TypeScript Version: 4.0
```
why you're limiting usage of this package? What's the purpose of this? (I cannot see TS 4.0 specific language feature being used. Thsi header restricts usage IMO

Reply:
My bad, copied the definition from another file which has limitations. All good to remove that line

Reply:
So please accept suggestion and remove the requirements, thx!